packages/table/src/regions.ts

```diff
@@ -513,10 +513,11 @@ export class Regions {
     }
 
     public static isRegionValidForTable(region: IRegion, numRows: number, numCols: number) {
-        if (region.rows != null && (region.rows[0] >= numRows || region.rows[1] >= numRows)) {
+        if (numRows === 0 || numCols === 0) {
             return false;
-        }
-        if (region.cols != null && (region.cols[0] >= numCols || region.cols[1] >= numCols)) {
+        } else if (region.rows != null && !intervalInRangeInclusive(region.rows, 0, numRows - 1)) {
+            return false;
+        } else if (region.cols != null && !intervalInRangeInclusive(region.cols, 0, numCols - 1)) {
```

Comment:
👍

Reply:
does `intervalInRangeInclusive` not do a null check?

Reply:
I could toss it in, but it felt clearer to not make the helper responsible for that (even though it leads to duplicated logic).